Modules/cjkcodecs/_codecs_iso2022.c

```diff
@@ -1119,18 +1119,19 @@ static const struct iso2022_designation iso2022_jp_ext_designations[] = {
 CONFIGDEF(jp_ext, NO_SHIFT | USE_JISX0208_EXT)
 
 
-BEGIN_MAPPINGS_LIST
+BEGIN_MAPPINGS_LIST(0)
   /* no mapping table here */
 END_MAPPINGS_LIST
 
-#define ISO2022_CODEC(variation) {              \
+#define ISO2022_CODEC(variation)                \
+st->codec_list[idx++] = (MultibyteCodec){       \
```

Comment:
I'd like to make one more change before landing. This `idx` manipulation is quite ugly. I would prefer to at least to hide it in a macro before landing this.

Reply:
I hid the ugly code in 65af11088e2d25b359ad81e88df8e6a5c668c969.